app.rb

```diff
@@ -1,118 +1,140 @@
-require 'json'
-require_relative './models/person'
-require_relative './models/student'
-require_relative './models/rental'
-require_relative './models/book'
-require_relative './models/teacher'
-require_relative './models/classroom'
-require_relative './models/data'
+require_relative './methods/create_person'
+require_relative './methods/create_book'
+require_relative './methods/create_rental'
+require_relative './load_data'
+require_relative './save_data'
 
 class App
-  include Presist
   def initialize
-    @books = load_books
-    @people = load_people
-    @rentals = load_rentals
+    @books = []
+    @students = []
+    @teachers = []
   end
 
-  def handle_action(option)
+  def display_options
+    print "
+      [1] List all books
+      [2] List all people
+      [3] Add a person
+      [4] Add a book
+      [5] Add a rental
+      [6] List all rentals for a given person id
+      [7] Exit
+      Select an option by typing its number: "
+  end
+
```

Comment:
* Kindly add unit tests for all the methods in all the classes:

> Book
> Person
> Student
> Teacher
> Classroom
> Rental
> all classes that you created while implementing the Decorator design pattern
* As I can not find any [Check this lesson for a refresher on unit testing](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/rspec_unit_test.md)

Reply:
well resolved! ✔️